README.md

```diff
@@ -16,12 +16,52 @@ A flexible JSON object linter with out of the box support for OpenAPI Specificat
 
 ## Installation
 
-```shell
+### API
+
+```bash
 npm install @stoplight/spectral
 ```
 
+### CLI
+
+```bash
+npm install -g @stoplight/spectral
+```
+
 Supports Node v8.3+ and modern browsers.
 
+### Executable binaries
+
+For users without Node and/or NPM/Yarn, we provide standalone packages for all major platforms:
+
+- x64 Windows
+- x64 MacOS
+- x64 Linux
+
+Note, the binaries are **not** auto-updatable, therefore you will need to download a new version on your own.
+
+Use of binaries is fairly easy.
+Once downloaded, you can proceed with the standard procedure for running any CLI tool.
+
+```bash
+./spectral-linux lint petstore.yaml
+```
+
+#### Installing binaries system-wide
+
+##### Linux
+
+As root, run:
```

Comment:
Do they need to be root? If so, should we put sudo in there?

Reply:
Yes, by default root permissions are required unless one changed it manually.
Added `sudo`.